src/webgpu/api/validation/createBindGroupLayout.spec.ts

```diff
@@ -63,27 +62,26 @@ g.test('maximum_binding_limit')
   `
   )
   .paramsSubcasesOnly(u =>
-    u //
-      .combine('binding', [
-        1,
-        4,
-        8,
-        256,
-        kLimitInfo.maxBindingsPerBindGroup.default - 1,
-        kLimitInfo.maxBindingsPerBindGroup.default,
-      ])
+    u.combine('bindingVariant', [1, 4, 8, 256, 'default', 'default-minus-one'])
   )
   .fn(t => {
-    const { binding } = t.params;
+    const { bindingVariant } = t.params;
     const entries: Array<GPUBindGroupLayoutEntry> = [];
 
+    const binding =
+      bindingVariant === 'default'
+        ? t.device.limits.maxBindingsPerBindGroup
+        : bindingVariant === 'default-minus-one'
+        ? t.device.limits.maxBindingsPerBindGroup - 1
+        : (bindingVariant as number);
```

Comment:
typescript doesn't realize this must be a number because we've handled the other cases above?

maybe need to say `[1, 4, 8, 256, 'default', 'default-minus-one'] as const` to make it pick it up?

Reply:
that was it! thanks. done.